lib/iex/lib/iex/autocomplete.ex

```diff
@@ -136,8 +136,16 @@ defmodule IEx.Autocomplete do
 
   defp expand_elixir_modules(list, hint) do
     mod = Module.concat(list)
-    format_expansion match_elixir_modules(mod, hint, list == []) ++
-                     match_module_funs(mod, hint), hint
+    match_elixir_modules(mod, hint, list == [])
+    |> Kernel.++(match_module_funs(mod, hint))
+    |> format_expansion(hint)
+  end
+
+  defp env_aliases() do
+    send(IEx.Server.whereis(), {:peek, self()})
+    receive do
+      {:peek, %IEx.State{env: env}} -> env.aliases
```

Comment:
I am worried about going into a situation where the server can't really reply but I can't think of any way it would happen. It would be nice to have a 5 seconds timeout just in case.